daemon/src/daemon.rs

```diff
@@ -5,8 +5,8 @@ use crate::peer;
 use crate::sandbox;
 use crate::types::{
     CursorState, EditorProtocolMessageError, EditorProtocolMessageFromEditor,
-    EditorProtocolMessageToEditor, EditorProtocolObject, EditorTextDelta, FileTextDelta,
-    JSONRPCFromEditor, JSONRPCResponse, PatchEffect, Range, RevisionedEditorTextDelta, TextDelta,
+    EditorProtocolMessageToEditor, EditorProtocolObject, FileTextDelta, JSONRPCFromEditor,
+    JSONRPCResponse, PatchEffect, Range, RevisionedEditorTextDelta, TextDelta,
```

Comment:
So many types, makes diff hard to read 😆 
Ok, basically we remove `EditorTextDelta`. Got it. :)